data/battle_4.s

```diff
@@ -7,44 +7,134 @@
 
 
 gMoveEffectBS_Ptrs:: @ 831C224
-	.incbin "baserom.gba", 0x31c224, 0xa4
-
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB386
+	.4byte gUnknown_082DB395
+	.4byte gUnknown_082DB3A4
+	.4byte gUnknown_082DB3B3
+	.4byte gUnknown_082DB3C9
+	.4byte gUnknown_082DB3E6
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3C2
+	.4byte gUnknown_082DB3D6
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3DD
+	.4byte gUnknown_082DB3F4
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3F4
+
+	.align 2
+	window_template 0x00, 0x01, 0x03, 0x07, 0x0f, 0x1f, 0x003f
+
+	.align 2
 gUnknown_0831C2C8:: @ 831C2C8
-	.incbin "baserom.gba", 0x31c2c8, 0x20
+	.incbin "graphics/battle_interface/unk_battlebox.gbapal"
 
+	.align 2
 gUnknown_0831C2E8:: @ 831C2E8
-	.incbin "baserom.gba", 0x31c2e8, 0xd8
+	.incbin "graphics/battle_interface/unk_battlebox.4bpp.lz"
 
-gUnknown_0831C3C0:: @ 831C3C0
-	.incbin "baserom.gba", 0x31c3c0, 0x18
+	.align 2
+	.byte 0x3A, 0x3E, 0x3B, 0x3F, 0x3C, 0x3D, 0x00, 0x00
+
+	.align 2
+gUnknown_0831C3B8:: @ 831C3B8
+	.byte 0x00, 0x00, 0x00, 0x80, 0x00, 0x00, 0x00, 0x00
 
+	.align 2
+gUnknown_0831C3C0:: @ 831C3C0
+	.2byte 0xd75a @ tileTag
+	.2byte 0xd75a @ paletteTag
+	.4byte gUnknown_0831C3B8
+	.4byte gDummySpriteAnimTable
+	.4byte NULL
+	.4byte gDummySpriteAffineAnimTable
+	.4byte sub_804F450
+
```

Comment:
define a macro for this?

Reply:
Is the intermediate step of a macro absolutely necessary? It's going to end up in a C struct anyway.